manifest-build-tools/application/reprove.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+# see http://stackoverflow.com/questions/1783405/checkout-remote-git-branch
+# and http://stackoverflow.com/questions/791959/download-a-specific-tag-with-git
+
+# check out a set of repositories to match the manifest file
+
+# Copyright 2015-2016, EMC, Inc.
+
+"""
+usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/reprove.py \
+--manifest manifest-files/rackhd-devel \
+--builddir d \
+--force \
+--git-credential https://github.com,GITHUB \
+--jobs 8 \
+checkout
+"""
```

Comment:
Please add some comments about those parameters.
for example, giving some examples for the last argument "action" .